app/src/main/java/com/example/newsapp/ui/home/fragments/EditProfileFragment.kt

```diff
@@ -37,24 +31,21 @@ class EditProfileFragment : Fragment() {
 
     private val editProfileViewModel: EditProfileViewModel by viewModels()
     private lateinit var binding: FragmentEditProfileBinding
+    private lateinit var filePath: Uri
     private lateinit var bitmap: Bitmap
-    private val bottomSheet = BottomSheetFragment()
-    private var isBottomSheetUp: Boolean = false
-
-    private val getImageFromCamera =
-        registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
-            if (result.resultCode == Activity.RESULT_OK) {
-                val photo = result.data?.extras?.get("data") as? Bitmap
-                photo?.let {
-                    saveImageToStorage(editProfileViewModel.getUserEmail(), it)
+    private val getImage =
+        registerForActivityResult(ActivityResultContracts.GetContent()) { result ->
+            if (result != null) {
```

Comment:
@jnandrawos  null checks can be replaced with safe calls

Reply:
Already done in other commits